perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -340,21 +352,46 @@ def Run(benchmark_spec):
 
   fio_command = 'sudo %s --output-format=json %s' % (fio.FIO_PATH,
                                                      REMOTE_JOB_FILE_PATH)
+
+  run_reps = FLAGS.run_for_minutes // MINUTES_PER_JOB
+  if FLAGS.run_for_minutes % MINUTES_PER_JOB != 0:
```

Comment:
Wow, this is bad. Thanks for catching it! I made the new logic run only for generated job files.

I thought about extending this flag to custom job files, but I don't see a good way to do it. We don't know how long a custom job file will last, so we can't figure out how many repeats we need. I could have made the flag number of repeats instead of minutes to run, but I think minutes are what we really care about - the number of minutes per job is just an implementation detail.

The only way I can see it working is if we start the custom job file and just keep running it until the total accumulated time passes the user's request, which might work. But I'm worried about measuring overhead somewhere in the system rather than actual fio run time, and fixing that would be very complex. I think that could be a separate project.